### PR TITLE
metrics: add basic resource control panels

### LINKED ISSUE #17595: Add resource control quota limit and wait duration panels
## Development Task

As title, adding them to tikv detail dashboard, so that we can easily notice if there is any throttle.

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed and how it works?

Issue Number: Close #17595

What's Changed:

```commit-message

```

### Check List

Tests 

- Manual test (add detailed scripts or steps below)

### Release note

```release-note
None
```

### CODE REVIEW

**Review by SpadeA-Tang — APPROVED** [review]

File: metrics/grafana/tikv_details.dashboard.py [hunk 1]
```diff
@@ -9287,6 +9287,40 @@ def StatusServer() -> RowPanel:
...
+    layout.row(
+        [
+            graph_panel(
+                title="Background Task Total Wait Duration",
+                yaxes=yaxes(left_format=UNITS.MICRO_SECONDS),
+                targets=[
+                    target(
+                        expr=expr_sum_rate(
+                            "tikv_resource_control_background_task_wait_duration",
+                            by_labels=["resource_group"],
```
Comment (glorv) [inline metrics/grafana/tikv_details.dashboard.py:9301]:
```suggestion
                            by_labels=["instance", "resource_group"],
```
Better by instance as this control is per-instance

File: metrics/grafana/tikv_details.dashboard.py [hunk 2]
```diff
@@ -9287,6 +9287,40 @@ def StatusServer() -> RowPanel:
...
+            ),
+            graph_panel(
+                title="Priority Quota Limit",
+                description="The memory usage of the resource control module.",
+                yaxes=yaxes(left_format=UNITS.MICRO_SECONDS),
+                targets=[
+                    target(
+                        expr=expr_sum(
+                            "tikv_resource_control_priority_quota_limit",
+                            by_labels=["priority"],
```
Comment (glorv) [inline metrics/grafana/tikv_details.dashboard.py:9314]:
```suggestion
                            by_labels=["instance", "priority"],
```

**Review by glorv — APPROVED** [review]

**Pull request merged**